js/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "surface",
   "version": "1.0.0",
   "description": "WIT IEEE Underwater ROV 2018 surface station",
-  "main": "ROV.js",
+  "main": "index.js",
   "scripts": {
     "test": "mocha test",
```

Comment:
Would it be better to run the method for detection on the pi or surface station?

Reply:
In the interest of processing power that would be a surface station thing.